devicemaps.json

```diff
@@ -97,7 +97,7 @@
           }
         ]
       },
-      "90500-0151-G53": {
+      "90500-0151": {
```

Comment:
Is there a way to decorate this JSON file to indicate that this product number reflects the Silicom Madrid series?

Reply:
We could add a field in this obj like `__comment__` or something that we wouldn't use/look at in the HWB, but I don't think we can just add `// comments` as it wouldn't be valid json then.

Reply:
we can just add a `description` field for something like this.